vespalog/src/main/java/com/yahoo/log/InitializeLog.java

```diff
@@ -0,0 +1,20 @@
+package com.yahoo.log;
+
+/**
+ * Sets up Vespa logging. Call a setup method to set up this.
+ *
+ * @author baldersheim
+ */
+public class InitializeLog {
+    static {
+        LogSetup.initVespaLogging("Container");
```

Comment:
The term "Container" seems a bit too specific for a generic module as vespalog.

Reply:
Agree, changed to 'unused-default'